portal/src/ReactApp.tsx

```diff
@@ -71,10 +73,13 @@ function isPosthogResetGroupsEvent(event: SentryErrorEvent) {
 }
 
 // DEV-1767: Unknown cause on posthog error, silence for now
-function sentryBeforeSend(event: SentryErrorEvent) {
+function sentryBeforeSend(event: SentryErrorEvent, hint: EventHint) {
   if (isPosthogResetGroupsEvent(event)) {
     return null;
   }
+  if (isNetworkError(hint)) {
```

Comment:
In authui we use `hint.originalException` but we use `hint` directly here. Why is the difference?

Reply:
Sorry it is a mistake, and as the type is `unknown` typecheck cannot catch it

Reply:
Fixed, thanks!